Makefile

```diff
@@ -17,7 +17,7 @@ build: pre-build
 	@packr2 clean
 
 release: pre-build
-	@go build -ldflags "-X $(CODEPATH)/cmd.version=$(RESUMIC_VERSION) -X $(CODEPATH)/cmd.gitCommit=$(GITCOMMIT)" -o resumic
+	@go build -ldflags "-s -w -X $(CODEPATH)/cmd.version=$(RESUMIC_VERSION)$(DIRTY_SUFFIX) -X $(CODEPATH)/cmd.gitCommit=$(GITCOMMIT)" -o resumic
```

Comment:
```suggestion
	@go build -ldflags "-s -w -X $(CODEPATH)/cmd.version=$(RESUMIC_VERSION) -X $(CODEPATH)/cmd.gitCommit=$(GITCOMMIT)" -o resumic
```

Reply:
Aha you're right, my bad! Fixed that in incoming commit.